doc/time_frequency.rst

```diff
@@ -22,6 +22,7 @@ Functions that operate on mne-python objects:
 .. autosummary::
    :toctree: generated/
 
+   compute_csd
```

Comment:
For consistency with `csd_fourier`, `csd_multitaper`, and `csd_morlet`, maybe `csd_tfr` would be better here?

Reply:
I'm not too particular about it, sounds good.